.gitattributes

```diff
@@ -1,2 +1,3 @@
+package-lock.json linguist-generated=true
 uv.lock linguist-generated=true
-web/yarn.lock linguist-generated=true
```

Comment:
Just to be clear, we can remove `web` here because simply specifying `package-lock.json` will match _any_ file with that name in the repo, correct?